tests/network/expose.go

```diff
@@ -107,11 +107,14 @@ var _ = SIGDescribe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:c
 	}
 
 	skipIfNotSupportedCluster := func(ipFamily ipFamily) {
+		if doesSupportIpv4(ipFamily) {
+			libnet.SkipWhenClusterNotSupportIpv4(virtClient)
+		}
```

Comment:
Isn't this redundant ? the `Skip` functions is already checking if cluster support it, so it should be just

```
libnet.SkipWhenClusterNotSupportIpv4(virtClient)
```